integration-tests/test_registration.py

```diff
@@ -40,7 +41,11 @@ def test_machineid_changes_on_new_registration(insights_client):
     with open(MACHINE_ID_FILE, "r") as f:
         machine_id_new = f.read()
 
-    assert machine_id_new != machine_id_old
+    if insights_client.core_version >= Version(3, 3, 13):
```

Comment:
I know you know this, but just for completion: the actual version needs to be changed before merging.

Reply:
fixed